includes/indices/class-algolia-index.php

```diff
@@ -716,13 +753,12 @@ abstract public function get_id();
 	 * @author WebDevStudios <contact@webdevstudios.com>
 	 * @since  1.0.0
 	 *
-	 * @param int   $page         The page.
-	 * @param int   $batch_size   The batch size.
-	 * @param array $specific_ids Array of IDs to retrieve and index.
+	 * @param int $page       The page.
+	 * @param int $batch_size The batch size.
 	 *
 	 * @return array
 	 */
-	abstract protected function get_items( $page, $batch_size, $specific_ids = [] );
+	abstract protected function get_items( $page, $batch_size );
```

Comment:
still need to get this edit reverted back to what's already in `main`

Reply:
Reverted.